app-main/public/js/events/serviceWorker.js

```diff
@@ -0,0 +1,13 @@
+// Register Service Worker
+export default async function registerSW() {
+  if ("serviceWorker" in navigator) {
+    try {
+      await navigator.serviceWorker.register("../../sw.js");
```

Comment:
Looks like we have some service worker caching issues. This will be due to the path change. I'd recommend moving all of the service worker registration logic back to `index.js` and reverting the registration path. You can see this errors in Chrome DevTools by toggling the network to offline and refreshing the page. Because the JS isn't cached the app fails to load.

![Screenshot 2021-01-07 at 10 51 11](https://user-images.githubusercontent.com/30320536/103885690-4bcfce00-50d8-11eb-9a00-c537531a19be.png)